content/en/platform/corda/5.0/deploying-operating/observability/metrics.md

```diff
@@ -230,3 +235,89 @@ the maximum value, and the cumulative sum of values, respectively.
+but also view it per network of virtual node.
+
+| Metric | Type | Tags | Description |
+| :----------- | :----------- | :----------- | :----------- |
+| `corda_membership_actions_handler_time_seconds` | Timer | <ul><li>`operation_name`</li><li>`group`</li><li>`virtualnode`</li></ul> | The time spent on membership actions. The `operation_name` tag is the name of the operation that the metric is related to. For example, `DistributeMemberInfo`, `StartRegistration`, `QueryMemberInfo`, and so on. The `group` tag is the membership group ID of the virtual node performing an operation or being monitored. It can also appear as `not_applicable` when a membership group identifier is not accessible when the metric was collected. The `virtualnode` tag is the virtual node short hash of the virtual node performing an operation or being monitored. This can also appear as `not_applicable` when a virtual node identifier was not accessible when the metric was collected.  |
+| `corda_membership_sync_handler_time_seconds` | Timer | <ul><li>`operation_name`</li><li>`group`</li><li>`virtualnode`</li></ul> | Measures how long it takes for each stage of synchronisation to complete. Synchronisation is split between different handler stages. It is processed on the MGM side and the network data package in constructed, and on the member side it is validated and persisted.  |
+| `membership_memberlist_cache_size` | Gauge | <ul><li>`group`</li><li>`virtualnode`</li></ul> | Gauge of the member list cache size to monitor how the cache size grows or shrinks. |
+
+#### Crypto Processor
+
+| Metric | Type | Tags | Description |
+| :----------- | :----------- | :----------- | :----------- |
+| `corda_crypto_flow_processor_execution_time_seconds` | Timer | <ul><li>`operation_name`</li></ul> | The time taken by crypto worker to process operations requested by flow operations. |
+| `corda_crypto_processor_execution_time_seconds` | Timer | <ul><li>`operation_name`</li></ul> | The time taken by crypto worker to process operations requested from other endpoints. |
+| `corda_crypto_wrapping_key_creation_time_seconds` | Timer | <ul><li>`tenant`</li></ul> | The time taken for wrapping key creation in crypto operations. The `tenant` tag is the identifier of a tenant: it's either a virtual node identifier or cluster level tenant ID. |
+| `corda_entity_manager_factory_creation_time_seconds` | Timer | <ul><li>`tenant`</li></ul> | The time taken to create entity manager factories. |
+| `corda_crypto_sign_time_seconds` | Timer | <ul><li>`signature_spec`</li></ul> | The time taken for crypto signing. The `signature_spec` identifies the signature signing scheme name to create signatures during crypto signing operations. |
+| `corda_crypto_sigining_key_lookup_time_seconds` | Timer | <ul><li>`lookup_method`</li></ul> | The time taken for crypto signing key lookup. The `lookup_method` tag indicates the method used to look up signing key hashes, either public key hashes or public key short hashes. |
```

Comment:
Can we please change "key hashes" to "key ids". The hash is just the implementation detail.